### PR TITLE
Updated year of copyright

### PR DESCRIPTION
**Pull request by RCMainak**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -178,7 +178,7 @@ <h1>{{ page.title }}</h1>
         <footer class="p-footer" role="contentinfo">
           <div class="p-content__row u-no-margin--left">
             <div class="col-12">
-              <p>&copy; 2018 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
+              <p>&copy; 2019 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
```
Comment (anthonydillon) [inline _layouts/default.html]:
```suggestion
              <p>&copy; {{ 'now' | date: "%Y" }} Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
```

**Comment by RCMainak** [comment]
Ops! I should given it bit more thought :sweat_smile:

**Comment by nottrobin** [comment]
@RCMainak don't worry about it. Simply flagging to us that this was out of date is already very helpful, filing a PR is even more so. Thank you so much for your help.

**Comment by nottrobin** [comment]
Looks good, works well. Thanks so much @RCMainak

**Pull request merged**